### ISSUE TITLE
Show net +1 - -1 votes on issue index

### ISSUE THREAD

**Issue by cirosantilli** [isaacs/github#605]
i.e. at https://github.com/isaacs/github/issues

Like the GitLab thumbs at: https://gitlab.com/gitlab-org/gitlab-ce/issues

**Comment by glen-84** [comment]
I don't think that negative values should be displayed, at least not until the negative value is significant (-5, -10, or some % of the average vote count).

If one random user is the first to vote, and down vote, the issue could easily become less visible, and influence other users.

**Comment by cirosantilli** [comment]
@glen-84 the same could be said for upvotes no? One user with no experience could do the upvote and give too much importance to something that should not have.

Maybe what you propose is a possibility. But the only real "solution" is more complicated algorithms, like a page rank / Reddit front page (considers time as well).

**Comment by glen-84** [comment]
Maybe, but I think that down votes are more damaging than up votes.

Perhaps there could be a minimum number of votes in either direction, before display.

**Comment by TPS** [comment]
It'd perhaps be more important to get the issue implemented first, & then suggest incremental improvements.  It's much less likely we'll receive improvements if we require huge effort on the front end.

**Comment by lamont-granquist** [comment]
:+1: on general idea and :+1: on only counting upvotes and not the net upvote-minus-downvote count (although it looks like configurability would be good -- but all I would want is upvotes and not downvotes).

and my rationale is that typically user downvotes aren't very high quality and are often biased based on some axe they want to grind or some philosophy they're afflicted with.  maintainers who typically have better quality 'negative' opinions that take into account the whole project and its technical limitations and goals already have the ability to close and lock threads for truly bad ideas.  on the other hand, a surplus of upvotes in some issues can be useful to inform maintainers of some pain point in their software that was in their blind spot.

clearly YMMV.

**Comment by ElectricRCAircraftGuy** [comment]
Taking it one step further, perhaps this should even be done by *net positive/negative reactions*. Ex: :+1: (positive reaction) + :heart:  = +2, and :-1:  + :frowning_face:  = -2, so something like :tada: + :+1:  + :heart:  + :frowning_face: + :-1:  = net +1
